common/pom.xml

```diff
@@ -44,6 +44,11 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>edu.illinois</groupId>
+                <artifactId>nondex-maven-plugin</artifactId>
```

Comment:
Please remove the change to add NonDex in your pull request. Developers are unlikely to add NonDex to their list of dependencies.